proposals/0207-rpc-message-protection.md

```diff
@@ -109,9 +109,9 @@ Mobile proxy shall send/receive encrypted RPC requests/responses if the app choo
 ### SDL core change
 
 #### Proposed solution
-Before the encryption of RPC service 7 is enabled, SDL rejects any RPC request with result code `ENCRYPTION_NEEDED` if the RPC needs protection. (Please see policy updates for which RPC needs protection.) SDL continues processing a RPC request if the RPC does not need protection. SDL sends a notification only if the notification RPC does not need protection.
+Before the encryption of RPC service 7 is enabled (encryption is not available), SDL rejects any RPC request with result code `ENCRYPTION_NEEDED` if the RPC needs protection. (Please see policy updates for which RPC needs protection.) SDL continues processing a RPC request if the RPC does not need protection. SDL sends a notification only if the notification RPC does not need protection.
 
-After the encryption of RPC service 7 is enabled, SDL rejects any unencrypted RPC request with result code `ENCRYPTION_NEEDED` with unencrypted response if the RPC needs protection. SDL continues processing an unencrypted RPC request if the RPC does not need protection and responds with unencrypted response. SDL continues processing an encrypted RPC request if the RPC needs protection and responds with encrypted response. SDL sends an unencrypted notification if the RPC does not need protection. SDL sends an encrypted notification if the RPC needs protection. In addition, SDL shall continue processing an encrypted RPC request if the RPC does not need protection and responds with encrypted response.
+After the encryption of RPC service 7 is enabled (encryption is available), SDL rejects any unencrypted RPC request with result code `ENCRYPTION_NEEDED` with unencrypted response if the RPC needs protection. SDL continues processing an unencrypted RPC request if the RPC does not need protection and responds with unencrypted response. SDL continues processing an encrypted RPC request if the RPC needs protection and responds with encrypted response. SDL sends an unencrypted notification if the RPC does not need protection. SDL sends an encrypted notification if the RPC needs protection. In addition, SDL shall continue processing an encrypted RPC request if the RPC does not need protection and responds with encrypted response.
```

Comment:
_If `SDL` is core please note it as `SDL Core`in all context_
Update to:

After the encryption of RPC service 7 is enabled (encryption is available), SDL rejects any unencrypted RPC request with result code `ENCRYPTION_NEEDED` with **the** unencrypted response, the RPC needs protection. SDL continues processing an unencrypted RPC request if the RPC does not need protection and responds with unencrypted response. SDL continues processing an encrypted RPC request if the RPC needs protection and responds with **an** encrypted response. SDL sends an unencrypted notification if the RPC does not need protection. SDL sends an encrypted notification if the RPC needs protection. In addition, SDL shall continue processing an encrypted RPC request if the RPC does not need protection and responds with encrypted response.

Reply:
updated